src/main/java/entities/login_leaderboard/User.java

```diff
@@ -6,16 +6,15 @@
  */
 public class User {
     private final String USERNAME;
-    private  String password;
-
+    private final String PASSWORD;
```

Comment:
Oh just one quick question; shouldn't we set this to not be final since we thought of having a password reset mechanism in place?